libs/openFrameworks/video/ofDirectShowPlayer.cpp

```diff
@@ -14,15 +14,14 @@
 //-------------------------------------------------------------------------------------------------------------------------------------------------------------
 
 
-#include <DShow.h>
+#include <dshow.h>
 #pragma include_alias( "dxtrans.h", "qedit.h" )
 #define __IDxtCompositor_INTERFACE_DEFINED__
 #define __IDxtAlphaSetter_INTERFACE_DEFINED__
 #define __IDxtJpeg_INTERFACE_DEFINED__
 #define __IDxtKey_INTERFACE_DEFINED__
-#include <uuids.h>
```

Comment:
If someone will be curious, [uuids.h is included in dshow.h already](https://github.com/ofTheo/videoInput/blob/master/videoInputSrcAndDemos/libs/DShow/Include/dshow.h#L51)